_data/simple-icons.json

```diff
@@ -7159,6 +7159,11 @@
             "hex": "00945E",
             "source": "https://www.moo.com/uk/about/press"
         },
+        {
+            "title": "Moonrepo",
+            "hex": "6F53F3",
+            "source": "https://raw.githubusercontent.com/moonrepo/moon/master/website/static/brand/icon-vector.svg"
```

Comment:
Can you change this to: `https://moonrepo.dev`

Reply:
resolved @dirien